### PR TITLE
Linux lsof: Add namespace dentry name

### PR DESCRIPTION
**Pull request by gcmoreira**
This PR adds namespace dentry name support.
Additionally, it improves the pointer address verification ref https://github.com/volatilityfoundation/volatility3/pull/1243.

Before:
```shell
$ python3 ./vol.py -r pretty \
	-f .../ubuntu180464bit_5.4.0-117.core \
	linux.lsof 
Volatility 3 Framework 2.10.0
  |  PID |         Process |  FD |                                                                Path |      Inode | Type |       Mode |                        Changed |                       Modified |                       Accessed |    Size
...
* |  770 |           lxcfs |   4 |                                                           /dev/fuse |         87 |  CHR | crw-rw-rw- | 2024-09-10 23:22:35.468000 UTC | 2024-09-10 23:22:35.468000 UTC | 2024-09-10 23:22:35.468000 UTC |       0
* |  770 |           lxcfs |   5 |                    <unsupported d_op symbol: ns_dname>:[4026532471] | 4026532471 |  REG | -r--r--r-- | 2024-09-10 23:22:37.936000 UTC | 2024-09-10 23:22:37.936000 UTC | 2024-09-10 23:22:37.936000 UTC |       0
* |  770 |           lxcfs |   6 |                                         /run/lxcfs/controllers/rdma |          1 |  DIR | dr-xr-xr-x | 2024-09-10 23:22:34.852000 UTC | 2024-09-10 23:22:34.852000 UTC | 2024-09-10 23:22:34.852000 UTC |       0
```

After:
```shell
$ python3 ./vol.py -r pretty \
	-f .../ubuntu180464bit_5.4.0-117.core \
	linux.lsof 
Volatility 3 Framework 2.10.0
  |  PID |         Process |  FD |                                                                Path |      Inode | Type |       Mode |                        Changed |                       Modified |                       Accessed |    Size
...
* |  770 |           lxcfs |   4 |                                                           /dev/fuse |         87 |  CHR | crw-rw-rw- | 2024-09-10 23:22:35.468000 UTC | 2024-09-10 23:22:35.468000 UTC | 2024-09-10 23:22:35.468000 UTC |       0
* |  770 |           lxcfs |   5 |                                                    mnt:[4026532471] | 4026532471 |  REG | -r--r--r-- | 2024-09-10 23:22:37.936000 UTC | 2024-09-10 23:22:37.936000 UTC | 2024-09-10 23:22:37.936000 UTC |       0
* |  770 |           lxcfs |   6 |                                         /run/lxcfs/controllers/rdma |          1 |  DIR | dr-xr-xr-x | 2024-09-10 23:22:34.852000 UTC | 2024-09-10 23:22:34.852000 UTC | 2024-09-10 23:22:34.852000 UTC |       0
```

### CODE REVIEW

**Review by ikelos — APPROVED** [review]
Ok, this seems good.  Do you want me to hold off merging this until #1263 gets merged (which now needs fixing up due to the change made in #1271), or merge this now and then remember the FIXME once #1263 goes in?

**Comment by gcmoreira** [comment]
Thanks @ikelos you can merge it. I will update #1263 soon

**Pull request merged**
